src/app/shared/post-it-collections/project-getter.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Subscription} from 'rxjs/Subscription';
+import {ProjectModel} from '../../core/store/projects/project.model';
+import {AppState} from '../../core/store/app.state';
+import {Store} from '@ngrx/store';
+import {OrganizationsAction} from '../../core/store/organizations/organizations.action';
+import {selectOrganizationByCode} from '../../core/store/organizations/organizations.state';
+import {ProjectsAction} from '../../core/store/projects/projects.action';
+import {selectProjectByCode} from '../../core/store/projects/projects.state';
+import {OrganizationModel} from '../../core/store/organizations/organization.model';
+
+export class ProjectGetter {
```

Comment:
We have always current organization and project in the store (look at `workspace.guard.ts`). So I think this class is needless.

Reply:
Hm.. I tried selecting them directly, without dispatching `GET`, but the subscribtion callback was never called.

I can try it out again later. I migh have tried wrong selectors (selectedProject didn't work for sure).

Reply:
Try to create new selector `selectCurrentOrganization` and `selectCurrentProject`. `selectedProject` works right only for workspace.

Reply:
Okay, thanks.

Reply:
I don't see much added value in this class at all. You just need to create the right selector in the store and it will be a one-liner to get the current project in a component.